api/app/__init__.py

```diff
@@ -1,2 +1,9 @@
-EMAIL="youremail@domain.com"
-PSWRD="password_goes_here"
\ No newline at end of file
+
+from dotenv import load_dotenv
+import os
+from fastapi import FastAPI  
+from decouple import config
+
+load_dotenv()
+CONTACT_EMAIL = os.getenv("EMAIL") 

```

Comment:
As i mentioned the environment variable itself should be developer friendly so developer understands it when he puts it in ENV.
`os.getenv("CONTACT_EMAIL")`
Code variables are good for maintainers to understand but user first.